pkg/virt-handler/vm.go

```diff
@@ -2425,8 +2425,8 @@ func (d *VirtualMachineController) helperVmShutdown(vmi *v1.VirtualMachineInstan
 		return err
 	}
 
-	// Only attempt to gracefully shutdown if the domain has the ACPI feature enabled
-	if isACPIEnabled(vmi, domain) && tryGracefully {
+	// Only attempt to gracefully shutdown if tryGracefully is true
+	if tryGracefully {
```

Comment:
can you add a unit test to make sure we don't have regressions on this?

Reply:
Done